src/js/components/keyword/KeywordPage.jsx

```diff
@@ -79,81 +75,27 @@ export default class KeywordPage extends React.Component {
-                formattedPrimeAmount =
-                    `${MoneyFormatter.formatMoneyWithPrecision(primeAmount / primeAmountUnits.unit, 1)}${primeAmountUnits.unitLabel}`;
-            }
-            else {
-                formattedPrimeAmount =
-                    `${MoneyFormatter.formatMoneyWithPrecision(primeAmount, 0)}`;
-            }
-        }
-
-        return (
-            <div className="keyword-header__summary">
-                <div className="keyword-header__summary-title">
-                    Search Summary
-                </div>
-                <div className="keyword-header__summary-award-amounts">
-                    <div className="keyword-header__summary-amount">
-                        Total Prime Award Amount: <span className="keyword-header__summary-amount keyword-header__summary-amount_bold">{formattedPrimeAmount}</span>
-                    </div>
-                </div>
-                <div className="keyword-header__summary-award-counts">
-                    <div className="keyword-header__summary-amount">
-                        Prime Award Transaction Count: <span className="keyword-header__summary-amount keyword-header__summary-amount_bold">{formattedPrimeCount}</span>
-                    </div>
-                </div>
-            </div>
-        );
-    }
-
     render() {
-        let searchSummary = null;
-        if (this.props.summary || this.props.summaryInFlight) {
-            searchSummary = this.generateSummary();
-        }
-
         return (
-            <div
-                className="usa-da-keyword-page">
-                <MetaTags {...MetaTagHelper.keywordPageMetaTags} />
-                <Header />
-                <StickyHeader>
-                    <div className="keyword-header">
-                        <div className="keyword-header__title">
-                            <h1>Keyword Search</h1>
-                        </div>
-                        {searchSummary}
-                        <div className="keyword-header__options">
-                            <DownloadButton
-                                disableHover={!this.props.keyword}
-                                downloadAvailable={this.props.downloadAvailable}
-                                onClick={this.clickedDownload} />
-                        </div>
-                    </div>
-                </StickyHeader>
+            <PageWrapper
+                classNames="usa-da-keyword-page"
+                title="Keyword Search"
+                metaTagProps={MetaTagHelper.keywordPageMetaTags}
+                toolBarComponents={[
+                    <SearchSummary
+                        primeAwardTotal={this.props.summary?.primeAmount}
+                        primeTransactionCount={this.props.summary?.primeCount}
+                        inFlight={this.props.summaryInFlight} />,
+                    <DownloadIconButton
+                        tooltipComponent={this.props.downloadAvailable
```

Comment:
The tooltip should not display before a user has entered a keyword (same on Advanced Search)
<img width="1601" alt="Screen Shot 2021-05-07 at 4 43 37 PM" src="https://user-images.githubusercontent.com/7108785/117509662-a549eb80-af58-11eb-9d1f-173c764d34a0.png">